pkg/depgraph/parsers/jsonl_output_parser_test.go

```diff
@@ -46,11 +46,11 @@ func TestJSONLOutputParser_ParseOutput_BlankOutput(t *testing.T) {
 func TestJSONLOutputParser_ParseOutput_InvalidJSON(t *testing.T) {
 	parser := NewJSONL()
 
-	input := []byte(`{"depGraph":{"invalid json},"normalisedTargetFile":"test"}`)
+	input := []byte(`This is a warning!`)
```

Comment:
Nit: I thought the previous fixture was a bit more obvious to understand what's going on. Should we use a multi-line string instead that illustrates how we parse valid JSON lines but skip invalid lines?